lib/taurus/qt/qtgui/graphic/taurusgraphic.py

```diff
@@ -867,12 +867,18 @@ class QGraphicsTextBoxing(Qt.QGraphicsItemGroup):
     _TEXT_RATIO = 0.8
 
     def __init__(self, parent=None, scene=None):
-        Qt.QGraphicsItemGroup.__init__(self, parent, scene)
-        self._rect = Qt.QGraphicsRectItem(self, scene)
+        Qt.QGraphicsItemGroup.__init__(self, parent)
+        if scene is not None:
+            scene.addItem(self)
+        self._rect = Qt.QGraphicsRectItem(self)
+        if scene is not None:
+            scene.addItem(self._rect)
         self._rect.setBrush(Qt.QBrush(Qt.Qt.NoBrush))
         self._rect.setPen(Qt.QPen(Qt.Qt.NoPen))
-        self._text = Qt.QGraphicsTextItem(self, scene)
-        self._text.scale(self._TEXT_RATIO, self._TEXT_RATIO)
+        self._text = Qt.QGraphicsTextItem(self)
+        if scene is not None:
+            scene.addItem(self._text)
+        # self._text.scale(self._TEXT_RATIO, self._TEXT_RATIO)
```

Comment:
This line is now commented... can it really be safely ignored?

Reply:
It was commented by mistake

Reply:
Sorry, it must be changed since it fails with py3